packages/codeeditor/src/editor.ts

```diff
@@ -424,6 +424,11 @@ export namespace CodeEditor {
      */
     setOption<K extends keyof IConfig>(option: K, value: IConfig[K]): void;
 
+    /**
+     * Set config options for the editor.
+     */
+    setOptions(options: any): void;
+
```

Comment:
This could be typed with 

```js
interface Options<K extends keyof IConfig>{
    K: IConfig[K];
}
setOptions<K extends keyof IConfig>(options: Options<K>[]): void;
```

Reply:
Included, thanks!